src/components/calendarComponent/calendar.scss

```
@@ -0,0 +1,219 @@
+@import '../../styles/mixins';
+
+.react-calendar { 
+  width: 809px;
+  max-width: 100%;
+  height: 555px;
+  max-height: 100%;
+  background-color: #fff;
+  color: #222;
+  border: none;
+  font-family: Arial, Helvetica, sans-serif;
+ }
+
+ .react-calendar button {
+  border: 1px solid rgb(238, 235, 235);
+}
+
+ .react-calendar__month-view__days {
+  border: 1px solid rgb(232, 230, 230);
+  box-shadow: 0 12px 24px rgba(0, 0, 0, 0.2);
+  border-radius: 8px;
+}
```

```suggestion
.react-calendar { 
  width: 809px;
  max-width: 100%;
  height: 555px;
  max-height: 100%;
  background-color: #fff;
  color: #222;
  border: none;
  font-family: Arial, Helvetica, sans-serif;
   & button{
       border: 1px solid rgb(238, 235, 235);
   }
   &__month-view__days{
        border: 1px solid rgb(232, 230, 230);
        box-shadow: 0 12px 24px rgba(0, 0, 0, 0.2);
        border-radius: 8px;
   }
 }
```
The big advantage of Sass is that you can make you code smaller and cleaner to read. Also you can make your other code like that. After transpilation this code you will have simple .css file.

---

Sure, I'll fix it